ipn/ipnlocal/serve.go

```diff
@@ -712,15 +720,20 @@ func (b *LocalBackend) addTailscaleIdentityHeaders(r *httputil.ProxyRequest) {
 	r.Out.Header.Del("Tailscale-User-Login")
 	r.Out.Header.Del("Tailscale-User-Name")
 	r.Out.Header.Del("Tailscale-User-Profile-Pic")
+	r.Out.Header.Del("Tailscale-Funnel-Host")
 	r.Out.Header.Del("Tailscale-Headers-Info")
 
 	c, ok := serveHTTPContextKey.ValueOk(r.Out.Context())
 	if !ok {
 		return
 	}
+	if c.Funnel != nil {
+		r.Out.Header.Set("Tailscale-Funnel-Host", c.Funnel.Host)
```

Comment:
you could just say `Tailscale-Funnel-Request=true` instead of returning the funnel host which callers won't really care about.

Reply:
ah, yeah, I was using `Tailscale-Funneled-Connection=?1` before for that reason, the `?1` is how you're supposed to [encode booleans in HTTP headers](https://httpwg.org/specs/rfc8941.html#boolean)

Reply:
sure that sounds fine too, sorry if we circled back to where you were :) The thing i mostly cared about was getting an accurate check of whether it was funnel or not.